.gitignore

```diff
@@ -3,3 +3,4 @@
 **/*.rs.bk
 Cargo.lock
 .criterion
+.vscode/
```

Comment:
this stuff belongs in your global gitignore file (see https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) 😉

Reply:
Thanks for pointing that out :)